src/connectors/menu/connectMenu.js

```diff
@@ -17,8 +17,8 @@ search.addWidget(
   customMenu({
     attribute,
     [ limit ],
-    [ showMoreLimit ]
-    [ sortBy = ['name:asc'] ]
+    [ showMoreLimit ],
+    [ sortBy = ['name:asc'] ],
```

Comment:
does it not have `refined, name:asc`?

Reply:
We usually don't provide the `isRefined` value to the array for a `menu`. But we probably should at the end. You can test with the value inside the stories, pick "Small Kitchen Appliances" the value will disappear from the list.